src/__tests__/version-test.ts

```diff
@@ -34,7 +32,7 @@ describe('Version', () => {
     expect(
       `${major}.${minor}.${patch}` +
         // istanbul ignore next (Can't be verified on all versions)
-        (preReleaseTag !== null ? '-' + preReleaseTag : ''),
+        (preReleaseTag !== null ? `-${preReleaseTag}` : ''),
```

Comment:
It's ok to leave it.
But as a general rule please keep repository wide PRs on a specific topic since it way harder to review and discussion of small changes will keep PR from being merged.